Tools/MGCB/BuildContent.cs

```diff
@@ -336,22 +337,33 @@ public void Build(out int successCount, out int errorCount)
                         message = ex.ContentIdentity.SourceFilename;
                         if (!string.IsNullOrEmpty(ex.ContentIdentity.FragmentIdentifier))
                             message += "(" + ex.ContentIdentity.FragmentIdentifier + ")";
-                        message += ": ";
+                        else
+                            message += " ";
                     }
-                    message += ex.Message;
+                    else
+                        message = c.SourceFile + " ";
+                    message += ": error ";
+
+                    // extract errorCode from MGFX message
+                    var match = Regex.Match(ex.Message, @"([A-Z]*[0-9]*):(.*)");
+                    if (match.Success || match.Groups.Count == 2)
+                        message += match.Groups[1].Value + " : " + match.Groups[2].Value;
+                    else
+                        message += ": " + ex.Message;
+
                     Console.WriteLine(message);
```

Comment:
Shouldn't this Console.Error.WriteLine as-well?

Reply:
This sound right.
I didn't change more code than I had to in order to make it work.
Yet, looking back at code I wrote a year ago I can't comment on anything! I am looking at it fresh.